doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -80,11 +80,16 @@ described elsewhere
 
 .. cmd:: Check @term
 
-   Displays the type of :n:`@term`. When called in proof mode, the
-   term is checked in the local context of the selected goal.
+   Displays the type of :n:`@term`. When called in proof mode, the term is
+   checked in the local context of the selected goal. This command tries to
+   resolve existential variables as much as possible. When resolved, the term
+   will be checked again.
```

Comment:
I don't know what to do with this last sentence. As is, it is probably insufficiently clear what that means to the reader and why it is so. So, without a better suggestion, I would suggest dropping it.